src/test/java/com/cloudant/tests/ViewsTest.java

```diff
@@ -266,12 +482,19 @@ public void paginationAscendingReusingViewAllTheWayInEachDirectionTwice() {
         checkPagination(false, 6, 2, view, 3, 1, 3, 1);
     }
 
+    @Test
```

Comment:
Add javadoc? Something like "Run the same check as the test paginationAscendingReusingViewAllTheWayInEachDirectionTwice on views using an array of multi-type keys."